lib/matplotlib/backend_tools.py

```diff
@@ -989,12 +989,10 @@ def trigger(self, *args, **kwargs):
                  'help': ToolHelpBase,
                  'copy': ToolCopyToClipboardBase,
                  }
-"""Default tools"""
```

Comment:
Are these not variable documentation strings that need to stay somehow?

Reply:
In the original PR it was decided that these were redundant with the variable name.